Platform/macOS/de.lproj/InfoPlist.strings

```diff
@@ -0,0 +1,6 @@
+/* Bundle name */
+"CFBundleName" = "UTM";
+
+/* (No Comment) */
+"UTM virtual machine" = "UTM Virtuelle Maschinen";
```

Comment:
This string describes what the `.utm` file extension is, so it needs to be singular. e. g. "UTM Virtuelle Maschine"